lib/vyasa_web/components/source_content/reading_content.ex

```diff
@@ -471,4 +471,36 @@ defmodule VyasaWeb.Content.ReadingContent do
     )
     |> assign(:kv_verses, mutated_verses)
   end
+
+  # Helper function that syncs and mutates draft table
+  defp mutate_draft_table(%{assigns: %{draft_table: %Vyasa.Sangh.Comment{} = dt, marks: marks}} = socket) do
+    IO.inspect(dt, label: "DROFT")
+    {:ok, com} = Vyasa.Sangh.update_comment(dt, %{marks: marks})
+    socket
+    |> assign(:draft_table, com)
+  end
+  # currently naive hd lookup can be filter based on active toggle,
+  # tree like comments can be used to store nested collapsible topics (personal mark collection e.g.)
+  # currently marks merged in and swapped out probably can be singular data structure
+  # if sangh_id is active open drafting table
```

Comment:
to be clarified @ sync later 

related to previous questions about marks being kept as lists or maps (seems like for the sake of a tree-structure)